packages/experimental/Avatar/src/__tests__/Avatar.test.jsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import { Avatar } from '..';
+import { getInitials } from '../useAvatar';
+import * as renderer from 'react-test-renderer';
+
+const alphabeticalTestNames = [
```

Comment:
I don't think getInitials is used by the NativeAvatar, so I think this file can be deleted?

Reply:
Same with the snap file